content/applications/hr/payroll/contracts.rst

```diff
@@ -0,0 +1,250 @@
+   :align: center
+   :alt: Optional tabs for a new contract.
+
+This section is where the specific salary details are defined. This section is country-specific, so
+depending on where the company is located, these fields will vary.
+
+Enter either the amount in the various fields, or check off a box to apply a benefit. Some options
+that can be entered here include *Meal Vouchers*, *Fuel Card*, *Internet*, *Paid Time Off*, etc.
+
+Some fields may be automatically filled in based off of the contracts selected in the *Contract
+Details* tab.
+
+Attachment of Salary
+~~~~~~~~~~~~~~~~~~~~
+
+.. image:: contracts/salary-attachment.png
+   :align: center
+   :alt: Optional tabs for a new contract.
+
+Any automatic deductions or allocations for an employee, such as child support payments and wage
+garnishments, are referred to as a *salary attachment*. This section is where all of these
+deductions or allocations are set.
+
+To add a new deduction, click on :guilabel:`Add a line`. Type in a description for the allocation
+under :guilabel:`Description`.
+
+.. image:: contracts/garnishment.png
+   :align: center
+   :alt: Enter a new line for each type of garnishment.
+
+Select the :guilabel:`Garnished Type` from the drop down. Chose from:
+
+- :guilabel:`Attachment of Salary` - Any payments taken out towards something that is *not* child
+  support. Typically any garnishments such as lawsuit payments, payments toward taxes owed, etc.
+- :guilabel:`Allocaiton of Salary` - Any deduction that is not required but voluntary, such as a
+  pre-tax allocation to a college savings account.
+- :guilabel:`Child Support` - Any payments taken out specifically for child support.
+
+Enter the start and end dates the entry applies to by clicking on the drop down menu under
+:guilabel:`From` and :guilabel:`To`, navigating to the correct month and year by using the arrow
+icons, then clicking on the date.
+
+Last, enter the *amount* that each payslip will pay towards the entry.
+
+Delete a line by clicking on the *trash can icon* at the end of the line.
+
+.. image:: contracts/delete-garnishment.png
+   :align: center
+   :alt: Remove a new line by clicking the delete icon.
+
+Save and send the contract
+--------------------------
+
+Once contracts have been created and/or modified, save the contract by clicking the :guilabel:`Save`
+smart button. Next, the contract must be sent to the employee to be signed.
+
+Click on one of the following smart buttons:
+
+.. image:: contracts/send-contract.png
+   :align: center
+   :alt: Send the contract to the employee via one of the smart buttons.
+
+- :guilabel:`Generate Simulation Link`: This option is for Belgian companies only. Clicking this
+  will open a pop-up that contains the basic information as well as a link for the contract when
+  using the slaary configurator. There is a :guilabel:`Send Offer` smart button that will send an
+  email to the employee so they can sign the contract.
```

Comment:
Thank you! This was a blind spot- so I dug into this. Apparently the contracts are all stored/sent/etc through the Documents app, and they are signed using the Sign app- so both apps must eb installed. Since the documents ap has written docs for that (so does sign) i mentioned this and linked to those docs since I didn't want to add a whole 'how to create/upload/manage/and sign' docs into this already long doc. But that took me a minute! The sign error appears when there is no signature field for the potential employee to sign in the contract being sent,.